content/articles/introduction-to-php-graphic-techniques/index.md

```diff
@@ -0,0 +1,227 @@
+ Introduction To PHP Graphic Techniques
+### Introduction
+PHP has the inherent ability to create and manipulate graphics independently. The scope and capabilities of PHP graphic handling is quite deep and powerful. It does this with the aid of the GD library which currently supports image formats in jpeg, png, gif and wbmp. 

```

Comment:
- This should be a standalone paragraph. Also, The second sentence should have some transitions to aid in article flow. For example, `....Therefore, the scope and capabilities`,.

-  Fix singular verb that doesn't agree with plural compound subject.
- Make good use of commas.
-  Quote unrecognised English words.